web-share/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
```

Comment:
> So the call to setupManualShareTest should be treated with the same degree of clarity as any ordinary function call in a piece of software,

There's abundant literature on how booleans in function signatures are a bad idea overall, e.g. Robert C. Martin in _Clean Code_:

> Passing a boolean into a function is a truly terrible practice. It immediately complicates the signature of the method, loudly proclaiming that this function does more than one thing.

So here's what I would suggest instead; create the two functions:

```js
setupManualShareTest({ title, url, text });
setupManualShareTestRequiringCancellation({ title, url, text }); // bikeshed name at will
```

This would make it a lot easier to review the tests, understand what they're doing, debug them and write new ones in the future.